pkg/replica/server.go

```diff
@@ -110,6 +110,7 @@ func (s *Server) Status() (State, Info) {
 		if os.IsNotExist(err) {
 			return Initial, Info{}
 		} else if err != nil {
+			logrus.Errorf("Read %s error %v", s.dir, err)
```

Comment:
logrus.Errorf("Failed to read info in replica directory %s: %v", s.dir, err)

Reply:
Thank u, comment addressed.